src/Shared/NativeMethodsShared.cs

```diff
@@ -44,9 +45,14 @@ internal static class NativeMethodsShared
         internal const int FILE_ATTRIBUTE_DIRECTORY = 0x00000010;
         internal const int FILE_ATTRIBUTE_REPARSE_POINT = 0x00000400;
 
+        private const int MAX_PATH_LEGACY_WINDOWS = 260;
```

Comment:
I'd leave this named `MAX_PATH` for uniformity. We can use it in a way that makes it optional.

Reply:
Right now it would conflict with the static int MAX_PATH already declared which can't be used as a const for the enum. But I think it would be safe to just change that to a const. It has a comment saying it's intended for unit tests to change the value, but no unit tests actually change it,.

Reply:
I guess we all know who's MAX_PATH this is, so no point calling it `WindowsLegacyMaxPath` :p